package.json

```diff
@@ -59,7 +59,7 @@
     "eslint-plugin-n": "^16.0.1",
     "eslint-plugin-promise": "^6.1.1",
     "jest-cli": "^29.6.2",
-    "marked": "^5.0.4",
+    "marked": "^4 || ^5 || ^6",
```

Comment:
This should be in the `peerDependencies`. The `devDependencies` version should be `v6.0.0`

Reply:
Sorry i kinda overlooked this.

Reply:
Fixed